python_ta/cfg/visitor.py

```diff
@@ -170,7 +170,7 @@ def visit_if(self, node: nodes.If) -> None:
 
         # Handle "then" branch and label it.
         then_block = self._current_cfg.create_block(
-            old_curr, edge_label="True", edge_condition=node.test
+            old_curr, edge_label="True", edge_condition=node.test, edge_negate=False
```

Comment:
Okay, so please remove the `edge_label` argument from this (and the other calls), so that we're only passing in `edge_negate`.

(This does mean that `edge_label` won't be used any more in our current code, but I'm thinking there will be opportunities to use it in the future.)

Reply:
Hi Professor, one problem with this update is that quite a few existing unit tests on CFG that checks for labels would fail. Would you want me to update these tests as well.

<img width="848" alt="image" src="https://github.com/user-attachments/assets/73f32fb8-e46e-44ce-8c10-d3241f7093d6">

Reply:
@Raine-Yang-UofT yeah this is interesting! This is a good time to add a getter method like `get_label()`, which could either use a (now-private) attribute `_label` or `negate` based on the logic I previously described. That way you just need to modify how the tests get labels (calling the method instead of accessing attributes directly) and you can use the same for the actual CFG generation.